ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/MainActivity.kt

```diff
@@ -184,13 +190,46 @@ fun WearApp(watchRepository: WatchRepository) {
                         onClickWatchList = {
                             swipeDismissableNavController.navigate(Screen.WatchList.route)
                         },
+                        onClickValueDisplay = {
+                            swipeDismissableNavController.navigate(Screen.ValueDisplay.route)
+                        },
                         proceedingTimeTextEnabled = showProceedingTextBeforeTime,
                         onClickProceedingTimeText = {
                             showProceedingTextBeforeTime = !showProceedingTextBeforeTime
                         }
                     )
                 }
 
+                composable(Screen.ValueDisplay.route) {
```

Comment:
Rename ValueDisplay to a more descriptive name to communicate it is a picker? for slider vs. stepper.